get-gemfile-deps

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
```

Comment:
We have quite the mix of python and ruby and bash scripts :) I think this is OK for now as long as we are heading towards one set of code.